### ISSUE TITLE
[files] Does not save paths

### ISSUE THREAD

**Issue by bambirombi** [albertlauncher/plugins#79]
After installing albert I added a path. It works even when I restart the computer. The problem is that since then, all the paths I add are not saved when I restart or log out. So I have to manually add all the paths (except the first one I added) every time I start the pc. 

This behavior started with the new version (0.18.7) and continues with the last one (0.18.10).

Albert version: 0.18.10
OS: Manjaro (Gnome)

**Comment by ManuelSchneid3r** [comment]
Do you get the "did not terminate graciously" message on start?

**Comment by bambirombi** [comment]
Yes: `[warn:albert] Application has not been terminated graciously`

**Comment by ManuelSchneid3r** [comment]
Then this problem should be solved first. Why does albert not shut down properly?

**Comment by bambirombi** [comment]
I uninstalled albert and deleted all the config files. Then I uninstalled from the arch [repo](http://aur.archlinux.org/packages/albert) instead of [OBS repo](https://software.opensuse.org/download.html?project=home:manuelschneid3r&package=albert). 

Previously it did not work installing albert from arch linux repo but now works and I can save new paths.

Moreover, the search box is centered at the top, so that problem has also been solved (issue #1018 of albert repo), at least for myself.

The warning continues to appear if I put `albert` on terminal, but it looks like all is working well.
